src/link/ping360simulationlink.cpp

```diff
@@ -45,5 +63,14 @@ void Ping360SimulationLink::randomUpdate()
     deviceData.updateChecksum();
     emit newData(QByteArray(reinterpret_cast<const char*>(deviceData.msgData), deviceData.msgDataLength()));
 
-    counter++;
+    // Calculate the global average time between requests
```

Comment:
This hunk (and related declarations above) belong with e845f3c031e5f0404e8e075eef4d564a3b95ec07

Reply:
Done.